File: panipaniRacing/Assets/Game 9/Script/Javascript/Game9_Cloud.js
	public var scrollSpeed = 0.5F;	//The speed the of the texture offset
	
    function Update()
	{
		//Make it smooth
        var offset = Time.time * scrollSpeed;
		
		//Set the texture offset
        renderer.material.mainTextureOffset = Vector2(offset, 0);
    }

File: panipaniRacing/Assets/Game 9/Script/Game9_Cloud.cs
using UnityEngine;
using System.Collections;

public class Game9_Cloud : MonoBehaviour
{
	public float scrollSpeed = 0.5F;	//The speed the of the texture offset
	
    void Update()
	{
		//Make it smooth
        float offset = Time.time * scrollSpeed;
		
		//Set the texture offset
        renderer.material.mainTextureOffset = new Vector2(offset, 0);
    }
}
